packages/jest-haste-map/src/watchers/WatchmanWatcher.js

```diff
@@ -35,7 +35,7 @@ export default function WatchmanWatcher(dir, opts) {
 }
 
 // eslint-disable-next-line no-proto
```

Comment:
The above eslint exclusion is not needed anymore?

Reply:
Sure, I'll update my PR later (I'm no more at home)

Reply:
I applied it, thanks!